src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -202,26 +203,27 @@ async function writeFile(fileHandle, contents) {
 }
 ```
 
-Note: There's no guarantee that the contents are written to disk until
-the `close()` method is called.
+To write data to disk, I needed a [`FileSystemWriter`][fs-writer]. Create one by
+calling `createWriter()` on the file handle object. If permission to write
```

Comment:
For additional emphasis, could we wedge in, between the two sentences here, something along these lines:

When `createWriter()` is called, Chrome first checks if the user has granted write permission to the file.